tests/sentry/discover/translation/test_mep_to_eap.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+
+from sentry.discover.translation.mep_to_eap import QueryParts, translate_mep_to_eap
+
+
+@pytest.mark.parametrize(
+    "input,expected",
+    [
+        pytest.param(
+            "",
+            "is_transaction:1",
+        ),
+        pytest.param(
+            "avg(transaction.duration):<10 OR transaction.duration:11",
+            "(avg(span.duration):<10 OR span.duration:11) AND is_transaction:1",
+        ),
+        pytest.param(
+            "has:transaction.duration OR has:measurement.lcp",
+            "(has:span.duration OR has:measurement.lcp) AND is_transaction:1",
+        ),
+        pytest.param(
+            "tags[foo]:10 OR transaction.duration:11",
+            "(tags[foo]:10 OR span.duration:11) AND is_transaction:1",
+        ),
+        pytest.param(
+            "(tags[foo]:10 OR transaction.duration:11) AND (p95(span.duration):<10)",
+            "((tags[foo]:10 OR span.duration:11) AND (p95(span.duration):<10)) AND is_transaction:1",
+        ),
```

Comment:
What happens if the input query already has `is_transaction:1`?

Reply:
Is there anything specific you're worried about